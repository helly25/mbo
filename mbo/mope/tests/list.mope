List example:
{{#list=[1,2,3,4]}}
{{list}}
{{/list}}

With escaped comma:
{{#list=[1\,2,3\,4]}}
{{list}}
{{/list}}

With joiner:
{{#list=[1\,2,3\,4];";"}}({{list}}){{/list}}
