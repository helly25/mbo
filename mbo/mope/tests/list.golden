List example:
1
2
3
4

With escaped comma:
1,2
3,4

With joiner:
(1,2);(3,4)
